using-timescaledb/continuous-aggregates.md

```diff
@@ -139,19 +139,67 @@ PostgreSQL. In addition, TimescaleDB continuous aggregates does not
 currently support the `FILTER` clause even though it is possible to
 parallelize but we might add support for this in a future version.
 
+#### Creating a continuous aggregate without refreshing it
```

Comment:
Do we really need a separate section for this? Ii seems quite repetitive and I think we can just  cover this in the previous section.  I mean, to some extent it already mentions it:  "When the view is created, it will (by default) be refreshed so that it..." We can just continue with "Add the WITH NO DATA option to avoid materializing the view on creation. This might be preferred when there's a lot of data to materialize or the continuous aggregate is created as part of transaction block."

Otherwise we are just repeating pretty much the same example over-and-over again.